be/src/olap/storage_engine.cpp

```diff
@@ -225,6 +226,35 @@ Status StorageEngine::_init_store_map() {
     for (auto store : tmp_stores) {
         _store_map.emplace(store->path(), store);
     }
+
+    auto st = _init_stream_load_record();
+    if (!st.ok()) {
+        LOG(WARNING) << "status=" << st.to_string();
+    }
+
+    return Status::OK();
+}
+
+Status StorageEngine::_init_stream_load_record() {
+    std::string stream_load_record_path;
+    if (_store_map.size() > 0) {
```

Comment:
Is it valid when _store_map is empty? Would it be better to explicit define the path in config file?